src/index.php

```diff
@@ -38,13 +41,16 @@
 
     // Send announcements
     if (isset($_POST["discord"])) {
-        send_discord_msg($title, $description . "\n\n" . $signature, $url);
+        send_discord_msg($title, $tag, $description . "\n\n" . $signature, $url);
     }
     if (isset($_POST["telegram"])) {
-        send_telegram_msg($title, $description . "\n\n" . $signature, $url);
+        send_telegram_msg($title, $tag, $description . "\n\n" . $signature, $url);
     }
     if (isset($_POST["email"])) {
-        $email_response = send_email_msg($title, $description, $signature, $reply_to, $url);
+        $email_response = send_email_msg($title, $tag, $description, $signature, $reply_to, $url);
+    }
+    if (isset($_POST["blankoweb"])) {
+        $email_response = send_web_msg($title, $description, $signature, $reply_to, $url);
```

Comment:
Assuming the tag isn't wanted to set to web, edited the comments above as such :)